tests/broker-engine/hosts-with-notes-and-actions.robot

```diff
@@ -172,4 +172,1185 @@ EBSN4
+    Ctn Engine Config Set Value In Hosts    0    host_1    gd2_image    gd2_image
+    Ctn Engine Config Set Value In Hosts    0    host_1    statusmap_image    statusmap_image
+    Ctn Engine Config Set Value In Hosts    0    host_1    initial_state    down
+    Ctn Engine Config Set Value In Hosts    0    host_1    check_interval    3
+    Ctn Engine Config Set Value In Hosts    0    host_1    retry_interval    2
+    Ctn Engine Config Set Value In Hosts    0    host_1    recovery_notification_delay    2
+    Ctn Engine Config Set Value In Hosts    0    host_1    max_check_attempts    8
+    Ctn Engine Config Set Value In Hosts    0    host_1    active_checks_enabled    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    passive_checks_enabled    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    event_handler_enabled    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    check_freshness    0
+    Ctn Engine Config Set Value In Hosts    0    host_1    freshness_threshold    14
+    Ctn Engine Config Set Value In Hosts    0    host_1    low_flap_threshold    53
+    Ctn Engine Config Set Value In Hosts    0    host_1    high_flap_threshold    15
+    Ctn Engine Config Set Value In Hosts    0    host_1    flap_detection_enabled    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    flap_detection_options    up
+    Ctn Engine Config Set Value In Hosts    0    host_1    notification_options    r
+    Ctn Engine Config Set Value In Hosts    0    host_1    notifications_enabled    0
+    Ctn Engine Config Set Value In Hosts    0    host_1    notification_interval    8
+    Ctn Engine Config Set Value In Hosts    0    host_1    first_notification_delay    6
+    Ctn Engine Config Set Value In Hosts    0    host_1    stalking_options    up
+    Ctn Engine Config Set Value In Hosts    0    host_1    process_perf_data    0
+    Ctn Engine Config Set Value In Hosts    0    host_1    2d_coords    50,50
+    Ctn Engine Config Set Value In Hosts    0    host_1    3d_coords    3.5,400,500
+    Ctn Engine Config Set Value In Hosts    0    host_1    obsess_over_host    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    retain_status_information    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    retain_nonstatus_information    1
+    Ctn Engine Config Set Value In Hosts    0    host_1    timezone    GMT+01
+    Ctn Engine Config Set Value In Hosts    0    host_1    severity_id    2
+    Ctn Engine Config Set Value In Hosts    0    host_1    icon_id    15
+    Ctn Engine Config Set Value In Hosts    0    host_1    _KEY1    VAL1
+    Ctn Engine Config Set Value In Hosts    0    host_1    _SNMPCOMMUNITY    pu
+    Ctn Engine Config Set Value In Hosts    0    host_1    _SNMPVERSION    2v
+    Ctn Engine Config Set Value In Hosts    0    host_1    group_tags    6
+    Ctn Engine Config Set Value In Hosts    0    host_1    category_tags    7
+
+    ${start}    Get Current Date
```

Comment:
You could have an issue here because this function rounds the timestamp. And it can be round with a greater value.
I think you should use Ctn Get Round Current Date that rounds the value with a smaller one.